assembl/static2/js/app/components/synthesis/actions.jsx

```diff
@@ -13,11 +13,11 @@ import EditPostButton from '../debate/common/editPostButton';
 import { getDiscussionSlug } from '../../utils/globalFunctions';
 import { DELETE_CALLBACK } from '../../constants';
 
-function userCanEdit(): boolean {
+export function userCanEdit(): boolean {
   return connectedUserCan(Permissions.EDIT_SYNTHESIS);
 }
 
-function userCanDelete(): boolean {
+export function userCanDelete(): boolean {
```

Comment:
shouldn't it be connectedUserCan(Permissions.DELETE_SYNTHESIS); instead ?

Reply:
we have one permission for create, edit and delete synthesis:  P_EDIT_SYNTHESIS